specification/redis/resource-manager/Microsoft.Cache/2017-10-01/redis.json

```diff
@@ -919,10 +919,10 @@
         "tags": [
           "Redis"
         ],
-        "operationId": "RedisLinkedServer_Create",
+        "operationId": "LinkedServer_Create",
```

Comment:
Breaking for SDK. On purpose?

Reply:
I prefer the new naming, but it's still breaking. Should we consider this Swagger as a preview Swagger?

Reply:
.net SDK never had this API. This is new features so its not breaking change.

Reply:
Note that C# is not the whole world :)
When a PR is merged to `current`, it means it's ready to be released for **all** SDK. Ruby, Node, Go, etc. This ApiVersion was merged 1 month ago, leaving room for SDK owner to use it.
FYI @salameer @marstr

Reply:
So, I checked: Java, Ruby, Python, Node are not using this ApiVersion yet, so this is fine. You tell me C# is fine as well. Go ships it already, but they are planning a breaking release anyway, so they will sync this together.
So we are fine for this breaking.

Reply:
There is a requirement to send swagger pull request for API review, before we can actually roll out it in RP. We are still in middle of rolling this into RP. What is a way to inform other teams i.e. java, go etc to note use something till it is out in all regions?